### DISCUSSION TITLE
Why can't I create a new discussion?

### DISCUSSION THREAD

**Discussion by tayllana** [community/community#116441] [Discussions]
### Select Topic Area

General

### Body

Why can't I create a new discussion? (The blue button) 
Mine looks like this:
![image](https://github.com/Murmele/Gittyup/assets/165088363/05bd2a8b-e5fb-4f55-b1ec-abe6f7a5a2c6)
but it should be like this:
![image](https://github.com/Murmele/Gittyup/assets/165088363/d0b33cbb-ca0d-4938-b088-86cb937607ca)

**Comment by queenofcorgis** [comment]
Do you have a new account? Per our [Temporary Interaction limits](https://github.com/orgs/community/discussions/48334), accounts must be older than 24 hours to make a post.

**Comment by LiteBrite82** [comment]
Is this still happening?

**Comment by Luuan11** [comment]
Hello, what's going on? 

On first impression, your account probably has less than 24 hours of use, if not, try confirming the email address of that account

Have a good week!

**Comment by NorthSystem** [comment]
There is a control at the organization level which can limit Discussions so that only those with triage+ permissions can open new discussions. Anyone can still comment on an existing/opened discussion.

If possible, I would recommend you reach out to the maintainers of this organization to see if they intended to have this control disabled.
